makemake/dirlist.cpp

```diff
@@ -240,7 +240,7 @@ bool DirList::matchString( ifstream &in, char &ch, const string &s ) {
       return false;
     }
   }
-  return in; // true if stream still good else false for no match
+  return true; // true if stream still good else false for no match
```

Comment:
I assume this is intended to (implicitly) call `operator bool()`, so this is probably correct:

```suggestion
  return !in.fail(); // true if stream still good else false for no match
```